config/opa-service/default_policies.ttl

```diff
@@ -0,0 +1,12 @@
+@prefix acl: <http://www.w3.org/ns/auth/acl#> .
+@prefix tenant: <https://tenant.url/> .
+
+tenant:policy1 a acl:Authorization ;
+    acl:agentClass <acl:agentClass:Admin> ;
+    acl:default <*> ;
```

Comment:
looking at the spec [3], it don't think using '*' make sense,
to me if the container is the service path, you would use a service path here,
and the base is going to be `/`.

also, most probably, as we serialize resources building on the base path included in the configuration,
the same should happens in the parsing, which implies that if the "base" path is not part of the resource url, we should throw an error.

but I am open to better ideas.

also, i noticed here [1] [2] that `acl:accessToClass` has be introduced, which basically is the resource type, right? in case we don't use this yet, we should.

references:
[1] - https://www.w3.org/ns/auth/acl
[2] - https://solid.github.io/web-access-control-spec/#authorization-extensions
[3] - https://github.com/solid/specification/issues/55